.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -1,16 +1,13 @@
 ---
 name: Bug report
-about: Create a bug report
-title: ''
+about: You have noticed a general issue or regression, and would like to report it
 labels: bug
-assignees: ''
-
 ---
 
 **Describe the bug**
 <!-- A clear and concise description of what the bug is. -->
 
-**To Reproduce**
+**Steps to Reproduce**
```

Comment:
Either all these titles should be title cased or none of them should. I'd vote for the former option.